fastlane/Fastfile

```diff
@@ -0,0 +1,84 @@
+# Customise this file, documentation can be found here:
+# https://github.com/fastlane/fastlane/tree/master/fastlane/docs
+# All available actions: https://github.com/fastlane/fastlane/blob/master/fastlane/docs/Actions.md
+# can also be listed using the `fastlane actions` command
+
+# Change the syntax highlighting to Ruby
+# All lines starting with a # are ignored when running `fastlane`
+
+# If you want to automatically update fastlane if a new version is available:
+# update_fastlane
+
+# This is the minimum version number required.
+# Update this, if you use features of a newer version
+fastlane_version "1.103.0"
+
+default_platform :ios
+
+platform :ios do
+  before_all do
+    ensure_xcode_version(version: "8.0")
+  end
+
+  desc "build the example project"
+  lane :build_example do |options|
+
+    cocoapods(
+      podfile: "Example/"
+    )
+
+    # cant use gym: https://github.com/fastlane/fastlane/issues/1816
+    xcodebuild(
+      workspace: "Example/PlayerKit.xcworkspace",
+      scheme: "PlayerKit-Example",
+      configuration: "Release",
+      build: true,
+      destination: "generic/platform=iOS",
+      build_settings: {
+        "CODE_SIGNING_REQUIRED" => "NO",
+        "CODE_SIGN_IDENTITY" => ""
+      },
+      buildlog_path: ".",
+      derived_data_path: "build/"
+    )
+
+  end
+
+  desc "run tests"
+  lane :test do |options|
+
+    cocoapods(
+      podfile: "Example/"
+    )
+
+    scan(
+      workspace: "Example/PlayerKit.xcworkspace",
+      scheme: "PlayerKit-Example",
+      configuration: "Debug",
```

Comment:
@MathiasGr lets switch this to Release. I expect when this is a cocoapod we'll want it to build with release settings.